easybuild/easyconfigs/r/Rcorrector/Rcorrector-1.0.6-GCC-11.3.0.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Rcorrector'
```

Comment:
@levlafayette you're not specifying a generic easyblock here, and we don't have an existing software-specific easyblock for `Rcorrector`

Reply:
The old version uses
```
easyblock = 'MakeCp'
```